content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md

```diff
@@ -64,7 +64,7 @@ For an example, consider a five-member etcd cluster running with the following c
 
 1. Run the following:
 
-       ./etcd --listen-client-urls=http://$IP1:2379, http://$IP2:2379, http://$IP3:2379, http://$IP4:2379, http://$IP5:2379 --advertise-client-urls=http://$IP1:2379, http://$IP2:2379, http://$IP3:2379, http://$IP4:2379, http://$IP5:2379
+   `./etcd --listen-client-urls=http://$IP1:2379, http://$IP2:2379, http://$IP3:2379, http://$IP4:2379, http://$IP5:2379 --advertise-client-urls=http://$IP1:2379, http://$IP2:2379, http://$IP3:2379, http://$IP4:2379, http://$IP5:2379`
```

Comment:
This needs to be properly fenced as a block, not just with syntactic highlighting. (Look at the raw text for this comment for specifics.)

Otherwise, the preview formats the text oddly (renders huge on mobile). 

```bash
./etcd --listen-client-urls=http://$IP1:2379, http://$IP2:2379, http://$IP3:2379, http://$IP4:2379, http://$IP5:2379 --advertise-client-urls=http://$IP1:2379, http://$IP2:2379, http://$IP3:2379, http://$IP4:2379, http://$IP5:2379
```